src/pages/guidelines/2x-grid/basics.mdx

```diff
@@ -101,7 +101,7 @@ When tiling fixed boxes, the column count is not known in advance, but a grid em
 
 #### Hybrid grid
 
-[Hybrid grids](#hybrid) are also frequent in practice, and hybrid boxes have properties of both.
+[Hybrid grids](#hybrid-grid) are also frequent in practice, and hybrid boxes have properties of both.
```

Comment:
```suggestion
Hybrid grids are also frequent in practice, and hybrid boxes have properties of both.
```

As far as I can tell, this is linking to its own header. I'd opt for just removing the link altogether.